DuckDuckGo/MainViewController.swift

```diff
@@ -1532,6 +1532,7 @@ extension MainViewController: AutoClearWorker {
             DaxDialogs.shared.resumeRegularFlow()
             self.forgetTabs()
         } onTransitionCompleted: {
+            ActionMessageView.present(message: UserText.actionForgetAllDone)
```

Comment:
FireButtonAnimation.swift#94 is showing this message when selecting the `None` option:

![tabs-and-data-cleared](https://user-images.githubusercontent.com/183774/117886516-30cac180-b264-11eb-89bf-a64488a45588.png)